memoized_person_or_org_data.txt

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"Tester, Mock"=>{:person_or_org=>{"type"=>"personal", "given_name"=>" Mock", "family_name"=>"Tester", "identifiers"=>[{:scheme=>"orcid", :identifier=>"1234-5678-9123-4567"}]}}, "User, Contributor"=>{:person_or_org=>{"type"=>"personal", "given_name"=>" Contributor", "family_name"=>"User"}, :role=>{:id=>"role-other"}}, "Laster, Firston"=>{:person_or_org=>{"given_name"=>"Firston", "family_name"=>"Laster", "type"=>"personal"}}, "Firston Laster"=>{:person_or_org=>{"name"=>"Firston Laster", "type"=>"organizational"}}, "Mckay, Frederick, S."=>{:person_or_org=>{"given_name"=>"Frederick S.", "family_name"=>"Mckay", "type"=>"personal"}}, "Miller, Alfred Frederick, Jr."=>{:person_or_org=>{"given_name"=>"Alfred Frederick Jr.", "family_name"=>"Miller", "type"=>"personal"}}, "Mason, Michael L., 1895-1963"=>{:person_or_org=>{"given_name"=>"Michael L., 1895-1963", "family_name"=>"Mason", "type"=>"personal"}}, "Unknown"=>{:person_or_org=>{"name"=>"Unknown", "type"=>"organizational"}}, "Creator not identified."=>{:person_or_org=>{"name"=>"Creator not identified.", "type"=>"organizational"}}, "Galter Health Sciences Library"=>{:person_or_org=>{"name"=>"Galter Health Sciences Library", "type"=>"organizational"}}, "Creator User"=>{:person_or_org=>{"type"=>"personal", "given_name"=>"Creator James", "family_name"=>"User"}}}
```

Comment:
This is done so that the values are replaced by the ones from the updated algorithm in production, right?

Reply:
Yes, exactly that